cmd/kspn/main.go

```diff
@@ -68,6 +68,22 @@ var spnHelpFlagGroups = []utils.FlagGroup{
 			nodecmd.ConfigFileFlag,
 		},
 	},
+	{
+		Name: "SERVICECHAIN",
+		Flags: []cli.Flag{
+			utils.ServiceChainConsensusFlag,
+			utils.ChildChainIndexingFlag,
+			utils.MainBridgeFlag,
+			utils.MainBridgeListenPortFlag,
+			utils.SubBridgeFlag,
+			utils.SubBridgeListenPortFlag,
+			utils.AnchoringPeriodFlag,
+			utils.SentChainTxsLimit,
```

Comment:
isn't it better adding `Flag` to the end of `SentChainTxsLimit`

Reply:
It is not directly related to this PR, so we will reflect it in future PR.